include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -247,95 +230,164 @@ struct TypeInfo;
+  *
+  * \param[in] domain NULL terminated string of the domain name
+  */
   explicit CustomOpDomain(const char* domain);
 
-  void Add(OrtCustomOp* op);
+  void Add(OrtCustomOp* op); ///< Adds an OrtCustomop to this domain object
 };
 
 struct RunOptions : Base<OrtRunOptions> {
-  RunOptions(std::nullptr_t) {}
-  RunOptions();
+  explicit RunOptions(std::nullptr_t) {} ///< \brief Create an empty RunOptions object, must be assigned a valid one to be used
+  RunOptions(); ///< \brief Create a new RunOptions object
 
-  RunOptions& SetRunLogVerbosityLevel(int);
-  int GetRunLogVerbosityLevel() const;
+  RunOptions& SetRunLogVerbosityLevel(int); ///< \brief Wraps OrtApi::RunOptionsSetRunLogVerbosityLevel
+  int GetRunLogVerbosityLevel() const; ///< \brief Wraps OrtApi::RunOptionsGetRunLogVerbosityLevel
 
-  RunOptions& SetRunLogSeverityLevel(int);
-  int GetRunLogSeverityLevel() const;
+  RunOptions& SetRunLogSeverityLevel(int); ///< \brief Wraps OrtApi::RunOptionsSetRunLogSeverityLevel
+  int GetRunLogSeverityLevel() const; ///< \brief Wraps OrtApi::RunOptionsGetRunLogSeverityLevel
 
-  RunOptions& SetRunTag(const char* run_tag);
-  const char* GetRunTag() const;
+  RunOptions& SetRunTag(const char* run_tag); ///< \brief wraps OrtApi::RunOptionsSetRunTag
+  const char* GetRunTag() const; ///< \brief Wraps OrtApi::RunOptionsGetRunTag
 
-  RunOptions& AddConfigEntry(const char* config_key, const char* config_value);
+  RunOptions& AddConfigEntry(const char* config_key, const char* config_value); ///< \brief Wraps OrtApi::AddRunConfigEntry
 
-  // terminate ALL currently executing Session::Run calls that were made using this RunOptions instance
+  /** \brief Terminates all currently executing Session::Run calls that were made using this RunOptions instance
+  *
+  * If a currently executing session needs to be force terminated, this can be called from another thread to force it to fail with an error
+  * Wraps OrtApi::RunOptionsSetTerminate 
+  */
   RunOptions& SetTerminate();
-  // unset the terminate flag so this RunOptions instance can be used in a new Session::Run call
+
+  /** \brief Clears the terminate flag so this RunOptions instance can be used in a new Session::Run call without it instantly terminating
+  *
+  * Wraps OrtApi::RunOptionsUnsetTerminate
+  */
   RunOptions& UnsetTerminate();
 };
 
+/** \brief Options object used when creating a new Session object
+*
+* Wraps ::OrtSessionOptions object and methods
+*/
 struct SessionOptions : Base<OrtSessionOptions> {
-  explicit SessionOptions(std::nullptr_t) {}
-  SessionOptions();
-  explicit SessionOptions(OrtSessionOptions* p) : Base<OrtSessionOptions>{p} {}
+  explicit SessionOptions(std::nullptr_t) {} ///< \brief Create an empty SessionOptions object, must be assigned a valid one to be used
+  SessionOptions(); ///< \brief Create a new SessionOptions object
+  explicit SessionOptions(OrtSessionOptions* p) : Base<OrtSessionOptions>{p} {} /// Used for interop with the C API
```

Comment:
>/// [](http://example.com/codeflow?start=80&length=3)

`///<`?

Reply:
See "Putting documentation after members" in Doxygen. Think of the < meaning 'applies to the statement to the left'